### PR TITLE
Jkmarx/registration redux upgrade

### LINKED ISSUE #2561: Registration Redux Upgrade
1.3 introduced a breaking change with register()
Seems worth spending the time to upgrade to 1.9 (last release before <1.11 support)

### PR DESCRIPTION
**Pull request by jkmarx**
Resolve #2561

### CODE REVIEW

File: refinery/core/views.py [hunk 1]
```diff
@@ -974,7 +974,7 @@ class of this backend as the sender.
                                      request=request)
         return new_user
 
-    def get_success_url(self, request, user):
+    def get_success_url(self, user):
```
Comment (mccalluc) [inline refinery/core/views.py]: Did the signature change between versions? Would a test that breaks under the old but works with the new be appropriate?

File: refinery/templates/core/about.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {% extends "base.html" %}
 {% load markup %}
-{% load flatblock_tags %}
```
Comment (scottx611x) [inline refinery/templates/core/about.html]: Are the `flatblocks` and `registration` apps intertwined somehow? Or are these just changes due to the `flatblocks` upgrade in `jkmarx/django-upgrade-1.8`

File: refinery/core/views.py (hunk 1 above)
Comment (jkmarx) [reply refinery/core/views.py]: We upgraded registration_redux from 1.1 to 1.9...the signature changed.

File: refinery/templates/core/about.html (hunk 2 above)
Comment (jkmarx) [reply refinery/templates/core/about.html]: They are not intertwined. I just didn't make another pull request for flatblocks. The changes are due to the apps upgrade.

**Pull request merged**